### DISCUSSION TITLE
country code not working

### DISCUSSION THREAD

**Discussion by alexQuestion** [ysfkaya/filament-phone-input#36] [General]
hello sir thanks for this perfect plugin but the countryStatePath('column_name_db') is not working for me and i try to get the data and didnt find it when using mutateDataBeforeSave()
when i delete countryStatePath it get the country code perfectly in the phone column 
what do you think the problem ??

**Comment by ranjeetmali** [comment]
Hi @alexQuestion and @ysfkaya ,
I'm encountering an issue with the PhoneInput component in Filament. Despite using the following code snippet:
```
PhoneInput::make('phone')
                ->label('Mobile Phone Number')
                ->showSelectedDialCode()
                ->countryStatePath('phone_country'),
```
Instead of displaying the country dial code, it's showing the country ISO code. Here's a screenshot illustrating the problem: 
![image](https://github.com/ysfkaya/filament-phone-input/assets/47209336/5ba94b7c-a951-45f1-a96b-c5ae1599368d)
I've reviewed the documentation but couldn't find a solution. Could you please assist me in resolving this issue? Any guidance or insights would be greatly appreciated.

Thank You!

**Reply by yaman3bd** [reply]
@ysfkaya 
I have the same issue, the `countryStatePath` uses the country ISO code instead of the country dial code.
any updates on this?
thanks
